protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -244,6 +244,20 @@ message ListCommitBlobsRequest {
     }
 }
 
+message FindBlobsRequest {
+    repeated uint64 repo_ids = 1; //if repo_ids not present in request then the backend returns all the matching repositories blobs, else returns a subsetfrom the list that are in the parameter repo_ids and return those repositories blobs
+    repeated string commits = 2; //if commits not present in request then the backend returns all the matching commits blobs, else returns a subsetfrom the list that are in the parameter commits and return those commits blobs
+    // List of folder names (with the last element potentially representing the blob name)
+    // This will be used to limit the search to scope just this location, like `git log /my/path` would do
+    repeated string location_prefix = 3;
+    repeated KeyValueQuery predicates = 4;
+
+    message Response {
+        repeated BlobExpanded blobs = 1;
```

Comment:
returning just BobExpanded wont be a lot of information. we should return https://github.com/VertaAI/modeldb/blob/4375484f4a24f13af610c804f3a3b0344592498e/protos/protos/public/modeldb/DatasetVersionService.proto#L83-L87

Reply:
I will create separate PR for this for all endpoints after finishing the basic modification